specs/lifespan.rst

```diff
@@ -46,14 +46,18 @@ Scope
 '''''
 
 The lifespan scope exists for the duration of the event loop. The
-scope itself contains basic metadata,
+scope itself contains basic metadata:
 
 * ``type``: ``lifespan``
 
 If an exception is raised when calling the application callable with a
-lifespan scope the server must continue but not send any lifespan
```

Comment:
I'm not sure this makes sense - how would you call the application with a ``lifespan.startup`` message?
Do I understand that any exceptions raised by an application handling the lifespan protocol should be ignored, with only the failed message counting?

Reply:
A server could reasonably either:

* Ignore the exception, log that it was disabling lifespan, and carry on as normal.
* Exit and inform the developer that they need to run disabling the lifespan protocol.

The important thing is that applications have an explicit way of communicating "The server attempted to startup and failed", which needs to be distinct from "I don't understand/support lifespan, I'd better raise an exception."

What's important here is that simple HTTP-only clients should be able to function gracefully, without the developer neccessarily having remembered to disable the lifespan protocol.